src/Consumer.cc

```diff
@@ -168,23 +166,13 @@ struct ConsumerNewInstanceContext {
     auto cConsumer = std::shared_ptr<pulsar_consumer_t>(rawConsumer, pulsar_consumer_free);
     auto listener = consumerConfig->GetListenerCallback();
 
-    if (listener) {
```

Comment:
We do not this after https://github.com/apache/pulsar-client-cpp/pull/447

Reply:
Could you open an issue for it?

Reply:
It has already been removed in this PR.

Reply:
On I see. I misunderstood it.